### DISCUSSION TITLE
如何更改高度？（How to change height?）

### DISCUSSION THREAD

**Discussion by fajianren** [Studio-42/elFinder#3596] [Q&A] [answered]
默认的高度是400px，我想改为100％。

我打开了js/**elfinder.full.js**，尝试改了里面的「**height = '400',**」和其他几处包括「400」的位置，但均无效。

请教，如何才能真正更改文件管理器界面的高度？
——————————————————
The default height is 400px and I want to change it to 100%.

I opened js/.elfinder.full.js and tried to change "height = '400'," and several other positions including "400", but none of them worked.

Please tell me, how can I actually change the height of the file manager interface?

**Comment by fajianren** [comment]
![86ade85268d4d9718b5aafc5f698626](https://github.com/Studio-42/elFinder/assets/103645753/eac3b96e-0edf-4c96-a515-852b55554310)

**Comment by RobiNN1** [comment]
There is `height` ( https://github.com/Studio-42/elFinder/wiki/Client-configuration-options-2.1#height ) config option.
Simply set `height: '100%',`.

Also, never edit elfinder.full.js/elfinder.min.js directly, it's a really terrible idea...

**Reply by fajianren** [reply]
![image](https://github.com/Studio-42/elFinder/assets/103645753/ef543f65-ff25-4364-974e-2603d1d98806)

I added "height: '100%'," to the "main.default.js" file, and then visited "elfinder.html" and "elfinder.legacy.html" again, and found that the height did not change. I have no experience with programming languages ​​and my approach may be wrong. Can you tell me the correct way to set "height: '100%',"? Thanks.
—————————
我在「main.default.js」文件中，增加了「height: '100%',」，然后再次访问「elfinder.html」「elfinder.legacy.html」，发现高度并没有变化。我没有编程语言的使用经验，我的方法可能有误。可以告诉我正确的设置「height: '100%',」的操作方法吗？谢谢。

**Reply by RobiNN1** [reply, marked as answer]
There is also a configuration in elfinder.html, set it here.

**Reply by fajianren** [reply]
我在根目录elfinder.html文件中的约142行的「defaultOpts : 」下一行增加了「heiaht:'100%',」，成功将高度更为了100％。谢谢。
——————————
I added "heiaht:'100%'," to the next line of "defaultOpts:" at line 142 of the elfinder.html file in the root directory, and successfully increased the height to 100%. Thanks.

**Reply by fajianren** [reply]
刚才说错了。更正如下：
我在根目录elfinder.html文件中的约17行的「defaultOpts : { 」下一行增加了「heiaht:'100%',」，成功将高度更为了100％。谢谢。
——————
I just said it wrong. the correct is:
I added "heiaht:'100%'," to the next line of "defaultOpts: {" about line 17 in the elfinder.html file in the root directory, and successfully increased the height to 100%. Thanks.
